ts/features/bonus/bpd/saga/orchestration/__tests__/activateBpdOnNewPaymentMethods.test.ts

```diff
@@ -7,12 +7,12 @@ import { navigateToWalletHome } from "../../../../../../store/actions/navigation
 import { appReducer } from "../../../../../../store/reducers";
 import { bpdRemoteConfigSelector } from "../../../../../../store/reducers/backendStatus";
 import { mockPrivativeCard } from "../../../../../../store/reducers/wallet/__mocks__/wallets";
-import { BpdConfig } from "../../../../../../types/backendStatus";
 import { EnableableFunctionsTypeEnum } from "../../../../../../types/pagopa";
 import { navigateToSuggestBpdActivation } from "../../../../../wallet/onboarding/bancomat/navigation/action";
 import { navigateToActivateBpdOnNewPrivative } from "../../../../../wallet/onboarding/privative/navigation/action";
 import { activateBpdOnNewPaymentMethods } from "../activateBpdOnNewAddedPaymentMethods";
 import { isBpdEnabled } from "../onboarding/startOnboarding";
+import { BpdConfig } from "../../../../../../../definitions/content/BpdConfig";
```

Comment:
Just to be clear: these are generated [io-services-metadata specs](from https://github.com/pagopa/io-services-metadata/pull/403/files#diff-f48ac1631a77bcfdff16e04799f86252e7f28d2f6ea1a7f4bc016097fd9710efR278). What is changing in this scope is not the returned value from the server but how the definitions are injected in the client.
I shouldn't require any special version of `io-dev-server` to work, correct?

Reply:
Yes you shouldn't.
These changes are only about type definitions (before manually, now automatically generated)

Note: this changes reflect the transitions of some fields from optional to required
see https://github.com/pagopa/io-services-metadata/pull/403#discussion_r667925554